### ISSUE TITLE
Apiary CLI doesn't work with ruby 2.4.1

### ISSUE THREAD

**Issue by netmilk** [apiaryio/apiary-client#159]
Apiary CLI doesnt work with ruby 2.4.1, and tests aren't passing. Just (bumping)[https://github.com/bblimke/webmock/issues/683] `rest-client` dependency to latest `2.0.2` `webmock`'s broken.
```
$ ruby -v
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-darwin16]
$ export APIARY_API_KEY=e4d...
$ apiary fetch --api-name myapi
/Users/netmilk/.rvm/gems/ruby-2.4.1/gems/rest-client-1.8.0/lib/restclient/request.rb:163:in `fetch': key not found: :ciphers (KeyError)
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/gems/rest-client-1.8.0/lib/restclient/request.rb:163:in `initialize'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `new'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/gems/rest-client-1.8.0/lib/restclient.rb:65:in `get'
	from /Users/netmilk/projects/apiary-client/lib/apiary/command/fetch.rb:57:in `query_apiary'
	from /Users/netmilk/projects/apiary-client/lib/apiary/command/fetch.rb:43:in `fetch_from_apiary'
	from /Users/netmilk/projects/apiary-client/lib/apiary/command/fetch.rb:27:in `execute'
	from /Users/netmilk/projects/apiary-client/lib/apiary/cli.rb:16:in `fetch'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/gems/thor-0.19.4/lib/thor/command.rb:27:in `run'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/gems/thor-0.19.4/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/gems/thor-0.19.4/lib/thor.rb:369:in `dispatch'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/gems/thor-0.19.4/lib/thor/base.rb:444:in `start'
	from /Users/netmilk/projects/apiary-client/bin/apiary:5:in `<top (required)>'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/bin/apiary:23:in `load'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/bin/apiary:23:in `<main>'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/netmilk/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by Ramblurr** [comment]
Will this get addressed? Ruby 2.4.1 was released 6 months ago.

**Comment by tu1ly** [comment]
https://github.com/apiaryio/apiary-client/pull/164
